docs-2.0/nebula-studio/about-studio/st-ug-what-is-graph-studio.md

```diff
@@ -4,24 +4,42 @@ Nebula Graph Studio (Studio in short) is a browser-based visualization tool to m
+
+- RPM-based. You can deploy Studio with RPM and connect it to Nebula Graph. For more information, see [Deploy Studio](../install-configure/st-ug-deploy.md).
+
+- Cloud Service. You can connect to Nebula Graph Cloud Service with one-click and create Nebula Graph database instance. For more information, see [Nebula Graph Cloud Service User Guide]([../install-configure/st-ug-deploy.md](https://cloud-docs.nebula-cloud.io/en/posts/toc/dbaas-ug-toc/)).
+
+The functions of the three releases are about the same. But because of different deployment methods, there will be different usage restrictions. For more information, see [Limitation](../about-studio/st-ug-limitations.md).
 
 ## Features
 
 Studio provides these features:
 
-- Graphical **Console** enables you to run nGQL statements and read the results in a human-friendly way.
+- Graphical user interface (GUI) makes Nebula Graph management more user-friendly:
+
+   - On the **Schema** page, you can manage schemas with a graphical user interface. It helps you quickly get started with Nebula Graph.
+
+   - On the **Console** page, you can run nGQL statements and read the results in a human-friendly way.
 
-- The **Explore** function enables you to explore the graph data. It helps you dig the relationships among data and improves the efficiency of data analysis.
+   - On the **Import** page, you can operate batch import of vertex and edge data with clicks, and view a real-time import log.
+
+- On the **Explore** page, you can explore the graph data. It helps you dig the relationships among data and improves the efficiency of data analysis.
 
 ## Scenarios
 
-You can use Studio v2.x in one of these scenarios:
+You can use Studio in one of these scenarios:
 
 - You have a dataset, and you want to explore and analyze data in a visualized way. You can use Docker Compose or Nebula Graph Cloud Service to deploy Nebula Graph and then use Studio to explore or analyze data in a visualized way.  
 
 - You have deployed Nebula Graph and imported a dataset. You want to use a GUI to run nGQL statements or explore and analyze graph data in a visualized way.  
 
 - You are a beginner of nGQL (Nebula Graph Query Language) and you prefer to use a GUI rather than a command-line interface (CLI) to learn the language.  
+
+## Authentication
+
+For Studio on Cloud, only the instance creator and the Nebula Graph Cloud Service accounts that are authorized to manipulate data in Nebula Graph can connect to Studio. For more information, see [Nebula Graph Cloud Service User Guide]([../install-configure/st-ug-deploy.md](https://cloud-docs.nebula-cloud.io/en/posts/toc/dbaas-ug-toc/)).
+
+For Docker-based and RPM-based Studio, authentication is not enabled in Nebula Graph by default and you can sign in to Studio with the default account and password (`user` and `password`) to log in to Studio.
```

Comment:
This sentence is too long. And there seems to be a logical issue: sign into Studio and log into Studio? Double-check it.